lib/glimesh/api.ex

```diff
@@ -58,11 +59,13 @@ defmodule Glimesh.Api do
   Parse a list of changeset errors into actionable API errors
   """
   def parse_ecto_changeset_errors(%Ecto.Changeset{} = changeset) do
-    Ecto.Changeset.traverse_errors(changeset, fn {msg, opts} ->
-      Enum.reduce(opts, msg, fn {key, value}, acc ->
-        String.replace(acc, "%{#{key}}", to_string(value))
-      end)
-    end)
+    changeset
+      |> Ecto.Changeset.traverse_errors(fn {msg, opts} ->
+           Enum.reduce(opts, msg, fn {key, value}, acc ->
+             String.replace(acc, "%{#{key}}", to_string(value))
+           end)
+         end)
+      |> Enum.map(fn {key, value} -> "#{key}: #{value}" end)
```

Comment:
The errors on a changeset are a map, but afaict absinthe requires error values to be either a single item or a list. It would throw up big error messages at me when I was testing locally.

Mapping the map into a list of strings solves this